### DISCUSSION TITLE
still can't speak in chinese

### DISCUSSION THREAD

**Discussion by adoal586** [JushBJJ/Mr.-Ranedeer-AI-Tutor#35] [General]
every time i want to communicate with GPT4 in chinese,it says:(Apologies, my ability to communicate in Chinese is limited. I will continue in English for a more accurate exchange)
I don't know why this happend,could you fix this issue?please

**Comment by JushBJJ** [comment]
I'm aware of this issue now, sorry for the inconvenience
https://github.com/JushBJJ/Mr.-Ranedeer-AI-Tutor/issues/37

**Reply by malixsys** [reply]
Should probably just be a non configurable rule…

**Comment by JushBJJ** [comment]
The issue is now be fixed in the latest version.
